components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/resources/devportal-api.yaml

```diff
@@ -2268,15 +2350,15 @@ paths:
         - OAuth2Security:
             - apim:app_import_export
       x-code-samples:
-        - lang: Shell
+        - lang: Curl
           source: 'curl -k -H "Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8"
-          "https://127.0.0.1:9443/api/am/devportal/v2/applications/import?appName=sampleApp&appOwner=admin&withKeys=true"
+          "https://localhost:9443/api/am/devportal/v2/applications/export?appName=sampleApp&appOwner=admin&withKeys=true"
```

Comment:
In all other examples, we used IP address instead of localhost.

Reply:
For devportal we have used localhost in all requests. But we have used the IP address for admin api, gateway api and publisher api. What should be the correct representation here?